template/src/utils/mobileWebTest.tsx

```diff
@@ -1,7 +1,7 @@
-const mobileAndTabletCheck = () => {
+const isMobileOrTablet = () => {
     let check = false;
```

Comment:
this shouldn't be a .tsx file and also just rename it isMobileOrTablet

Reply:
Done